src/backend/web/validation.js

```diff
@@ -50,4 +51,41 @@ function validateNewFeed() {
   };
 }
 
-module.exports.validateNewFeed = validateNewFeed;
+// query validation starts here
+// queryValidation rules
+const queryValidationRules = () => {
+  return [
+    // text must be between 1 and 256
+    check('text')
+      .exists()
+      .withMessage('text should not be empty')
+      .isLength({ max: 256, min: 1 })
+      .withMessage('text should be between 0 to 256 characters'),
+    // filter must exists
```

Comment:
exist vs. exists

Reply:
Incorporated in [7dd761b](https://github.com/Seneca-CDOT/telescope/pull/1334/commits/7dd761bc7b9281cd31a572a30424e0c41235cb6b)